server/docs/game.yaml

```diff
@@ -139,3 +139,63 @@ paths:
                   value:
                     success: false
                     error: 'Player does not exist'
+
+  /game/{gameId}:
+    patch:
+      tags:
+        - 'Game'
+      summary: 'Update the dates and location of a game.'
+      description: 'This can only be done by a logged in user and a league admin.'
+      parameters:
+      - name: 'gameId'
+        in: 'path'
+        description: 'ID of the game.'
+        required: true
+        type: 'string'
+      security:
+      - BearerAuth: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/GameUpdate'
```

Comment:
Right now this references a GameUpdate schema with properties like team1 and team2, but technically we want a schema with properties like location, date etc.

Maybe you could look at `season.yaml` and copy something similar for something like GameChange and change this schema to that?

Reply:
Okay I made changes. Mind checking if it's alright now?

Reply:
oops didnt see that joseph reviewed this pr before, so will resolve this

Reply:
I guess an object is more accurate for the `newLocation` type, but it doesn't matter since your example is correct so as long as it uses that one it should be fine! Nice work